src/firefly/js/visualize/ui/MouseReadoutOptionPopups.jsx

```diff
@@ -46,27 +70,89 @@ function getDialogBuilder(fieldKey) {
 
 }
 
-export function showMouseReadoutOptionDialog(fieldKey) {
+export function showMouseReadoutOptionDialog(fieldKey,radioValue) {
 
-	getDialogBuilder(fieldKey);
+	getDialogBuilder(fieldKey, radioValue);
 	AppDataCntlr.showDialog(fieldKey);
 }
 
-function showSelectedField(request, groupKey, fieldKey){
-	console.log('closing ' + groupKey);
+/**
+ *
+ * This method map the value in coordinate option popup to its value
+ * @param coordinateRadioValue : the value in the radio button
+ * @returns {{coordinate: *, type: *}}
+ */
+export function getCoordinateMap(coordinateRadioValue){
+	var coordinate;
+	var type;
+	switch (coordinateRadioValue) {
+		case 'eqj2000hms':
+			coordinate = CoordinateSys.EQ_J2000;
+			type = 'hms';
+			break;
+		case 'eqj2000DCM':
+			coordinate = CoordinateSys.EQ_J2000;
+			type = 'decimal';
+			break;
+		case'galactic':
+			coordinate = CoordinateSys.GALACTIC;
+			type = null;
+			break;
+		case 'eqb1950':
+			coordinate = CoordinateSys.EQ_B1950;
+			type = null;
+			break;
+
+		case 'pixelSize':
+			coordinate = CoordinateSys.PIXEL;
+			break;
+		case 'sPixelSize':
+			coordinate = CoordinateSys.SCREEN_PIXEL;
+			break;
+		default:
+			coordinate=CoordinateSys.UNDEFINED;
+			break;
+
+
+
+	}
```

Comment:
I notice that is function is only use in MouseReadout, not MouseReadoutOptionPopups. You should probably move it to there so you don't have to export it.